### DISCUSSION TITLE
Custom Source based on multi-sites

### DISCUSSION THREAD

**Discussion by davidhellmann** [craftcms/cms#13344] [Ideas] [answered]
When creating custom sources (great feature, like it a lot), I have the problem that the custom sources are visible on all multi-sites, and also if the sections are not enabled for that, they're visible. That's a bit of bad behavior and really confusing to the authors when there are entry sources on the left but nothing it's inside.

Is there a way to handle that or is that a FR? :)

**Comment by MoritzLost** [comment]
+1 
Would be great to be able to restrict custom sources to a specific site, as far as I can tell that's not possible without hooks.

**Comment by brandonkelly** [comment, marked as answer]
Good idea! Added for 4.5 (#13368).

**Reply by davidhellmann** [reply]
Thanks Brandon!

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out with this 🎉
